distributor-node/src/services/networking/NetworkingService.ts

```diff
@@ -65,6 +67,12 @@ export class NetworkingService {
     this.downloadQueue.on('error', (err) => {
       this.logger.error('Data object download failed', { err })
     })
+
+    this.initRuntimeApi().catch((err) => this.logger.error('Runtime API initialization failed:', { err }))
```

Comment:
> we should ensure the request handler catches it and return a HTTP 503 status rather than cause the express server to cause the process to exit with an unhandled expection.

I don't think this is currently happening for `distributor-node` (yes, `storage-node` is crashing), if `getQueryNodeStatus()` throws, then it is being caught by the express [error handler](https://github.com/zeeshanakram3/joystream/blob/master/distributor-node/src/services/httpApi/HttpApiBase.ts#L92) middleware.

Reply:
> if getQueryNodeStatus() throws, then it is being caught by the express error handler middleware.

Was this added recently? Referencing my previous comment:

> Getting {"type":"exception","message":"Unexpected end of JSON input"} from /status when squid is not available

Reply:
Hmm actually it is handled and Argus doesn't crash, so I guess we just need better error message in that case